StortSpelProjekt/Engine/src/Renderer/Texture/Texture.cpp

```diff
@@ -42,4 +42,19 @@ TEXTURE_TYPE Texture::GetType() const
 const unsigned int Texture::GetDescriptorHeapIndex() const
 {
 	return m_pSRV->GetDescriptorHeapIndex();
-}
\ No newline at end of file
+}
+
+BYTE* Texture::GetData() const
+{
+	return m_pImageData;
+}
+
+unsigned int Texture::GetWidth()
+{
+	return m_ResourceDescription.Width;
+}
+
+unsigned int Texture::GetHeight()
+{
+	return m_ResourceDescription.Height;
+}
```

Comment:
Add const, unless you want people to be able to change these?

Reply:
As I am giving a unsigned int and not a reference or pointer for that matter there is no way for someone to change the values in the class.